### PR TITLE
fix: fix verify-token predicate values check

### PR DESCRIPTION
**Pull request by asraa**
Signed-off-by: Asra Ali <asraa@google.com>

One more fix: https://github.com/slsa-framework/slsa-github-generator/actions/runs/4055979822/jobs/6979852459

systemParamters, at least right now, have "values".

(this may change later with v1.0)

### CODE REVIEW

**Review by laurentsimon — APPROVED** [review]

**Comment by laurentsimon** [comment]
@ianlewis `pre-submit e2e docker-based default / build / provenance (pull_request)` is failing

**Comment by asraa** [comment]
Yes -- also don't merge yet, I was testing this and the jq var sub needs those jq variables declared

File: .github/workflows/scripts/e2e-utils.sh [hunk 1]
```diff
@@ -93,7 +93,7 @@ e2e_verify_predicate_v1_buildDefinition_buildType() {
 }
 
 e2e_verify_predicate_v1_buildDefinition_systemParameters() {
-    _e2e_verify_query "$1" "$3" '.buildDefinition.systemParamters.'"$2"
+    _e2e_verify_query "$1" "$3" '.buildDefinition.systemParamters."$2".value'
```
Comment (ianlewis) [inline .github/workflows/scripts/e2e-utils.sh:96]: > Expressions don't expand in single quotes, use double quotes for that.

Yeah, this needs to be fixed. Also parameters was misspelled.
```suggestion
    _e2e_verify_query "$1" "$3" ".buildDefinition.systemParameters.\"$2\".value"
```

**Review by ianlewis — APPROVED** [review]

File: .github/workflows/scripts/e2e-utils.sh (hunk 1 above)
Comment (asraa) [reply .github/workflows/scripts/e2e-utils.sh]: thank you!

**Pull request merged**